testsuite/integration/src/test/scripts/manualmode-build.xml

```diff
@@ -105,6 +105,12 @@
         <copy todir="target/jbossas-dwm-manual-1" overwrite="true">
             <fileset dir="target/wildfly"/>
         </copy>
+        <copy todir="target/jbossas-custom" overwrite="true">
+            <fileset dir="target/wildfly"/>
+        </copy>
+        <copy todir="target/jbossas-custom/standalone/configuration" overwrite="true">
+            <fileset dir="src/test/resources/org/jboss/as/test/manualmode/model/custom-configurations"/>
```

Comment:
@istraka Are all the files in this dir basically meant to be what we ship in docs/examples/configs, or have they been altered in some way to fit the needs of the test?

I think it's best if as much as possible we test the configs we provide in docs/examples/configs, as those are what will be maintained; copies can fall out of date. There are different ways that can be accomplished.

It's also possible to configure arquillian to directly use the file in docs/examples/configs. See https://issues.redhat.com/browse/WFARQ-75

Reply:
IIRC I took them from 73 zip, so they are bit different. Yeah, we should not maintain it. Now they are taken `build/docs/...` automatically.  `standalone-picketlink.xml` however needs some changes so that `attribute` nodes in LDAP store can be set to an expression. Hence the xslt script.

Moreover, in order to use readOnlyServerConfig I must change to `jboss.server.config.file.name` property `config.file.name` due to [1]. Otherwise Arquillian throws exception since both `serverConfig` and `readOnlyServerConfig` are used.

[1] https://github.com/wildfly/wildfly-arquillian/blob/master/container-managed/src/main/java/org/jboss/as/arquillian/container/managed/ManagedContainerConfiguration.java#L33

Reply:
See the latest changes.